core/cloudflow-build-support/src/main/scala/com/lightbend/cloudflow/buildtool/KafkaSupport.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.cloudflow.buildtool
+
+import java.time.Duration
+import java.util.UUID
+import java.util.concurrent.atomic.AtomicReference
+
+import org.testcontainers.{ utility => tcutility }
+import org.testcontainers.containers.KafkaContainer
+import org.testcontainers.containers.wait.strategy.Wait
+
+import scala.util.Try
+import scala.collection.JavaConverters._
+
+object KafkaSupport {
+
+  val KafkaPort = 9093
+  val kafka = new AtomicReference[KafkaContainer]()
+
+  def setupKafka(debug: String => Unit) = {
+
+    val cl = Thread.currentThread().getContextClassLoader()
+    val kafkaPort =
+      try {
+        val c = getClass().getClassLoader()
+        Thread.currentThread().setContextClassLoader(c)
+
+        val k = new KafkaContainer(tcutility.DockerImageName.parse("confluentinc/cp-kafka:5.4.3"))
```

Comment:
Maybe externalize the image name ?

Reply:
Can the `parse` fail / throw exceptions ?

Reply:
not with a fixed string, if it succeeds once it succeed always I assume